src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -1013,194 +1013,196 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
       }
     }
 
-    "WorkspaceService PFB Tests" - {
-      "should 400 if PFB URL is empty" in {
-        (Post(pfbImportPath, HttpEntity(MediaTypes.`application/json`, """{"url":""}"""))
-          ~> dummyUserIdHeaders(dummyUserId)
-          ~> sealRoute((workspaceRoutes)) ~> check {
-            status should equal(BadRequest)
-          })
-      }
-
-      "should 400 if PFB URL is not https" in {
-        (Post(pfbImportPath, HttpEntity(MediaTypes.`application/json`, s"""{"url":"http://missing.avro"}"""))
-          ~> dummyUserIdHeaders(dummyUserId)
-          ~> sealRoute(workspaceRoutes)) ~> check {
-            status should equal(BadRequest)
-          }
-      }
```

Comment:
removed some tests because we now delegate this logic to the import service; instead, we have a test to ensure we bubble up whatever errors the import service might return.